.github/workflows/auto-merge.yml

```diff
@@ -0,0 +1,18 @@
+name: "Dependabot auto-merge"
+
+on: pull_request
+
+permissions:
+  contents: write
+  pull-requests: write
+
+jobs:
+  dependabot:
+    runs-on: ubuntu-latest
+    if: ${{ github.actor == 'dependabot[bot]' }}
+    steps:
```

Comment:
We can also extract some metadata about prs and potentially fully auto-merge some PRs
```suggestion
    steps:
     - name: Dependabot metadata
        id: metadata
        uses: dependabot/fetch-metadata@v1.1.1
        with:
          github-token: "${{ secrets.GITHUB_TOKEN }}"
```

Reply:
There's this metadata lurking in the commit Dependabot creates that is extracted by the action <img width="662" alt="Screenshot 2022-12-15 at 11 25 27" src="https://user-images.githubusercontent.com/20165/207847528-1c575099-425a-4f01-81b2-d20eddc050d3.png">